lib/active_merchant/billing/gateways/d_local.rb

```diff
@@ -155,11 +159,22 @@ def parse_house_number(address)
 
       def add_card(post, card, action, options = {})
         post[:card] = {}
+        if card.is_a?(NetworkTokenizationCreditCard)
+          post[:card][:network_token] = card.number
+          post[:card][:cryptogram] = card.payment_cryptogram
+          post[:card][:eci] = card.eci
+          # used case of Network Token: 'CARD_ON_FILE', 'SUBSCRIPTION', 'UNSCHEDULED_CARD_ON_FILE'
+          post[:card][:stored_credential_type] = options[:stored_credential_type] if options[:stored_credential_type]
+          # required for MC debit recurrent in BR 'FIRST'(subsecuence Payments) . 'USED' an inital payment
```

Comment:
this comment seems reversed? shouldn't "first" be an initial payment?

Reply:
yes, sorry, I have to fix it.